### ISSUE TITLE
Add p(0) + p(1) = X as a metric

### ISSUE THREAD

**Issue by niznik-dev** [niznik-dev/predicting-zygosity#57]
# Describe the Feature

This ticket is specifically to implement the p(0) + p(1) custom metric. I learned when working on this that:

- The list of tensors that is returned from the model is 8 subdivisions of the full sequence length (in this case, 29 each for a total of 29 * 8 = 232 tokens)
- Because our input is always 216 tokens long (for the fake twin data), we can hardcode that we want the logits at position 216 (but this is a terrible hardcode so we need a better way of attacking this longterm)

## Related Issues

#18 

# Acceptance Criteria
- [ ] During any test run of finetuning with an existent `finetune_custom_metrics.py`, this metric (name to be finalized) appears on the WandB output
- [ ] We develop a way to determine the sequence length without hardcoding (i.e. there is code that can determine 216 for this test case and is reliable if the input sequence varies - test with capitalization dataset?)

**Comment by niznik-dev** [comment]
I should discuss this one with @drigobon because I think some of the tools he's created could help clean this up before merging (especially RE: hardcoding logit position)
